create-vm.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -euo pipefail
+
```

Comment:
@jlebon Absolutely. I have added that in the blog post I'm doing for fedora magazine and projectatomic. Certainly will add the same here :). Thanks for your workshop and the script. Got the inspiration from you to work on this.